manifests/o/OpenArena/OpenArena/0.8.8/OpenArena.OpenArena.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with komac v2.10.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: OpenArena.OpenArena
+PackageVersion: 0.8.8
+InstallerType: zip
+NestedInstallerType: portable
+ArchiveBinariesDependOnPath: true
+NestedInstallerFiles:
+  - RelativeFilePath: openarena-0.8.8/openarena.exe
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+UpgradeBehavior: uninstallPrevious
+ReleaseDate: 2018-02-05
+Installers:
+  - Architecture: neutral
+    InstallerUrl: https://www.moddb.com/downloads/start/40515
+    InstallerSha256: 3CF76A033DE958A596841DFC9C048DC407DC863F4070301EB36F27AE89F98A4C
```

```suggestion
    InstallerSha256: 5A8FAF7F5B51F351B0A1618C06B6B98A5F1A6758F1D39818DE2C87DF2A0BAC4A
```

That is the correct sha256 checksum of the archive.